Allow for Log4J to be configured for peons but still ensure console logging is enforced (#14094)
apache/druid: docs/configuration/logging.md

> > NOTE:

Reviewer: ```suggestion
```

> > Although Druid shares the log4j configuration file task peon processes,

Reviewer: Shares from whom to whom? Should it say "shares the log4j configuration file **between** task peon processes"?

Author: How is this for the write up?
>Although Druid shares the log4j configuration file amongst all services, including task peon processes, the appenders in this file WILL NOT take effect for loggers in the peon processes unless there is a console appender defined. Instead all the appenders for peons processes will be overwritten to be console appenders and use the log levels as defined in the configuration file. This means you can configure loggers at different logging level for task logs using log4j2.xml.

I think keeping the note is useful since it is something that isn't intuitive, but you know docs better than me so i'm happy to remove it it

> > the appenders in this file WILL NOT take effect for peon processes if there is no console appender defined.

Reviewer 2: ```suggestion
> the appenders in this file WILL NOT take effect for loggers in the peon processes unless there is a console appender defined.
```

> Peons always output logs to standard output. Middle Managers redirect task logs from standard output to 

> > [long-term storage](index.md#log-long-term-storage).

Reviewer: This information seems more of a statement of fact; consider moving outside the note block. I think it also interrupts the idea flow to log-level settings in the following lines.
